### PR TITLE
fix: include metadata cols in get dataframe

### PR DESCRIPTION
**Pull request by elboy3**
https://app.shortcut.com/galileo/story/8671/dq-get-dataframe-doesn-t-support-metadata

See also changes in API https://github.com/rungalileo/api/pull/1488

### CODE REVIEW

File: dataquality/metrics.py [hunk 1]
```diff
@@ -305,6 +308,8 @@ def get_dataframe(
         (embeddings, probabilities etc), vaex will always be returned, because pandas
         cannot support multi-dimensional columns. Default True
     :param include_data_embs: Whether to include the off the shelf data embeddings
+    :param meta_cols: List of metadata columns to return in the dataframe. If "*"
+        is passed, return all metadata columns
```
Comment (bogdan-galileo) [inline dataquality/metrics.py:312]: Is it that the list has to contain the string `*` ?

Comment (elboy3) [reply dataquality/metrics.py:312]: Yeah, so ["*"] would include all metadata

Comment (bogdan-galileo) [reply dataquality/metrics.py:312]:
```suggestion
        is included, return all metadata columns
```

**Review by bogdan-galileo — APPROVED** [review]
LGTM

**Review by setu4993 — APPROVED** [review]

**Pull request merged**